libs/langchain/langchain/llms/symblai_nebula.py

```diff
@@ -69,19 +56,19 @@ class Config:
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key and python package exists in environment."""
         nebula_service_url = get_from_dict_or_env(
-            values, "nebula_service_url", "SYMBLAI_NEBULA_SERVICE_URL"
+            values, "nebula_service_url", "NEBULA_SERVICE_URL"
```

Comment:
you can pass a default value as a fourth param here (so you don't have to manually do it later

Reply:
changes made!